R/tooltip.R

```diff
@@ -0,0 +1,125 @@
+#' Add a tooltip to a UI element
+#'
+#' Tooltips are useful for showing additional information when focusing (or
+#' hovering over) a UI element.
+#'
+#' @param trigger A UI element (i.e., [htmltools tag][htmltools::tags]) to serve
+#'   as the tooltips trigger. It's good practice for this element to be a
+#'   keyboard-focusable and interactive element (e.g., `actionButton()`,
+#'   `actionLink()`, etc) so that the tooltip is accessible to keyboard and
+#'   assistive technology users.
+#' @param ... UI elements for the tooltip. Character strings are automatically
+#'   [htmlEscape()]d unless marked as [HTML()].
```

Comment:
```suggestion
#' @param ... UI elements for the tooltip. Character strings are
#'   [automatically escaped][htmlEscape()] unless marked as [HTML()].
```

and should the links be `[htmltools::htmlEscape()]` and `[htmltools::HTML()]`?